backend/scraper/scrape_heb.py

```diff
@@ -70,6 +70,29 @@ def parseLink(self, url):
             image_link = None
             pass
 
+        #Get the products quantity / net weight
+        try:
+            quantity = soup.find('div', {'class': 'pdp-mobile-title-price-section'}) #specific div container for item count could not be parsed, instead used parent div container 

```

Comment:
Also, looks like this PR includes the changes from #47. With Git, it's better to keep the changes separate, so I'm going to do a little bit of rebasing magic before you merge this into master